server/scripts/importer/import.js

```diff
@@ -84,6 +84,7 @@ async function parseRecord(record, argv) {
     voter_id: record[0],
     first_name: record[3],
     last_name: record[4],
+    age: record[13],
```

Comment:
I have not dealt with these CSVs before; it seems like 13 would be the last column based on context.

Reply:
Ah sounds like this is actually formatted as date of birth (much better for integrity but just a bit more complex to import). I should probably get a CSV sample before updating anything further.

This work is on hold while focusing on new priorities but I'm happy to wrap it up any time.